### PR TITLE
Enhance API, REST, ping documentation.

### LINKED ISSUE #2944: REST API is not working
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

I would like to use REST API to on-the-fly reconfiguration.

### What did you expect to see?

REST API is accessible and can configure Traefik with it

### What did you see instead?

Every API call ends with error 404

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
1.5.3
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
loglevel = "DEBUG"

[api]
dashboard = true

[rest]
entryPoint = "traefik"
```
API call from clients:
```
curl -XPUT "http://lb-tester.tryadmins.net:8080/api" -d '{ "frontends" : { "frontend1" : {"backend" : "backend1" } }, "backends" : { "backend1" : { "servers" : { "server1" : { "URL" : "www.google.com" } } } } }'
404 page not found
```

### If applicable, please paste the log output in debug mode (`--debug` switch)

```
INFO[2018-02-27T16:46:30+01:00] Using TOML configuration file /root/traefik_rest.toml 
INFO[2018-02-27T16:46:30+01:00] Traefik version v1.5.2 built on 2018-02-12_10:56:31AM 
INFO[2018-02-27T16:46:30+01:00] 
Stats collection is disabled.
Help us improve Traefik by turning this feature on :)
More details on: https://docs.traefik.io/basics/#collected-data
 
DEBU[2018-02-27T16:46:30+01:00] Global configuration loaded {"LifeCycle":{"RequestAcceptGraceTimeout":0,"GraceTimeOut":0},"GraceTimeOut":0,"Debug":false,"CheckNewVersion":true,"SendAnonymousUsage":false,"AccessLogsFile":"","AccessLog":null,"TraefikLogsFile":"","TraefikLog":null,"LogLevel":"DEBUG","EntryPoints":{"http":{"Network":"","Address":":80","TLS":null,"Redirect":null,"Auth":null,"WhitelistSourceRange":null,"Compress":false,"ProxyProtocol":null,"ForwardedHeaders":{"Insecure":true,"TrustedIPs":null}},"traefik":{"Network":"","Address":":8080","TLS":null,"Redirect":null,"Auth":null,"WhitelistSourceRange":null,"Compress":false,"ProxyProtocol":null,"ForwardedHeaders":{"Insecure":true,"TrustedIPs":null}}},"Cluster":null,"Constraints":[],"ACME":null,"DefaultEntryPoints":["http"],"ProvidersThrottleDuration":2000000000,"MaxIdleConnsPerHost":200,"IdleTimeout":0,"InsecureSkipVerify":false,"RootCAs":null,"Retry":null,"HealthCheck":{"Interval":30000000000},"RespondingTimeouts":null,"ForwardingTimeouts":null,"Web":null,"Docker":null,"File":null,"Marathon":null,"Consul":null,"ConsulCatalog":null,"Etcd":null,"Zookeeper":null,"Boltdb":null,"Kubernetes":null,"Mesos":null,"Eureka":null,"ECS":null,"Rancher":null,"DynamoDB":null,"ServiceFabric":null,"Rest":{"EntryPoint":"traefik","CurrentConfigurations":null},"API":{"EntryPoint":"traefik","Dashboard":true,"Debug":false,"CurrentConfigurations":null,"Statistics":null},"Metrics":null,"Ping":null} 
INFO[2018-02-27T16:46:30+01:00] Preparing server http &{Network: Address::80 TLS:<nil> Redirect:<nil> Auth:<nil> WhitelistSourceRange:[] Compress:false ProxyProtocol:<nil> ForwardedHeaders:0xc4202c2080} with readTimeout=0s writeTimeout=0s idleTimeout=3m0s 
INFO[2018-02-27T16:46:30+01:00] Preparing server traefik &{Network: Address::8080 TLS:<nil> Redirect:<nil> Auth:<nil> WhitelistSourceRange:[] Compress:false ProxyProtocol:<nil> ForwardedHeaders:0xc4202c20c0} with readTimeout=0s writeTimeout=0s idleTimeout=3m0s 
INFO[2018-02-27T16:46:30+01:00] Starting server on :80                       
INFO[2018-02-27T16:46:30+01:00] Starting provider *rest.Provider {"EntryPoint":"traefik","CurrentConfigurations":{}} 
INFO[2018-02-27T16:46:30+01:00] Starting server on :8080                     
^CINFO[2018-02-27T16:50:46+01:00] I have to go... interrupt                    
INFO[2018-02-27T16:50:46+01:00] Stopping server gracefully                   
DEBU[2018-02-27T16:50:46+01:00] Waiting 0s seconds before killing connections on entrypoint traefik... 
DEBU[2018-02-27T16:50:46+01:00] Waiting 0s seconds before killing connections on entrypoint http... 
DEBU[2018-02-27T16:50:46+01:00] Entrypoint traefik closed                    
DEBU[2018-02-27T16:50:46+01:00] Entrypoint http closed                       
INFO[2018-02-27T16:50:46+01:00] Server stopped                               
INFO[2018-02-27T16:50:46+01:00] Shutting down
```

### PR DESCRIPTION
**Pull request by ldez**
### What does this PR do?

- Add or move configuration examples.
- fix REST call example

### Motivation

Enhance API, REST, ping documentation.

### More

- [x] Added/updated documentation

### Additional Notes

Fixes #2944

### CODE REVIEW

File: docs/configuration/api.md [hunk 1]
```diff
@@ -62,7 +62,101 @@ For more customization, see [entry points](/configuration/entrypoints/) document
...
+  [entryPoints.http]
+  address = ":80"
+
+  [entryPoints.foo]
+  address = ":8080"
+
+  [entryPoints.bar]
+  address = ":8081"
+
+# Activate API and Dashboard
```
Comment (mmatur) [inline docs/configuration/api.md]: You need to explicitly ask with property `dashboard = true` to activate dashboard

Comment (ldez) [reply docs/configuration/api.md]: fixed

File: docs/configuration/api.md [hunk 2]
```diff
@@ -62,7 +62,102 @@ For more customization, see [entry points](/configuration/entrypoints/) document
     For compatibility reason, when you activate the rest provider, you can use `web` or `rest` as `provider` value.
     But be careful, in the configuration for all providers the key is still `web`.
 
-### Provider configurations
+### Address / Port
+
+You can defined a custom address/port like that:
```
Comment (mjbright) [inline docs/configuration/api.md]: define

Comment (mjbright) [reply docs/configuration/api.md]: You can define a custom address/port like this:

File: docs/configuration/api.md [hunk 3]
```diff
@@ -62,7 +62,102 @@ For more customization, see [entry points](/configuration/entrypoints/) document
...
+* Regular path: `http://hostname:80/foo`
+* Admin Panel: `http://hostname:8083/`
+* Ping URL: `http://hostname:8082/ping`
+
+In the above example, it is _very_ important to create a named dedicated entry point, and do **not** include it in `defaultEntryPoints`.
+Otherwise, you are likely to expose _all_ services via that entry point.
+
+### Custom Path
+
+You can defined a custom path like that:
```
Comment (mjbright) [inline docs/configuration/api.md]: You can define a custom path like this:

File: docs/configuration/api.md [hunk 4]
```diff
@@ -62,7 +62,102 @@ For more customization, see [entry points](/configuration/entrypoints/) document
...
+[ping]
+entryPoint = "foo"
+
+[api]
+entryPoint = "bar"
+```
+
+In the above example, you would access a regular path, administration panel, and health-check as follows:
+
+* Regular path: `http://hostname:80/foo`
```
Comment (psalaberria002) [inline docs/configuration/api.md]: `foo` is confusing here since it's also the name of an entryPoint

**Pull request merged**